src/v2/guide/instance.md

```diff
@@ -1,24 +1,27 @@
 ---
-title: The Vue Instance
+title: Permintaan Vue
 type: guide
 order: 3
 ---
 
-## Creating a Vue Instance
+## Membuat Permintaan Vue
 
-Every Vue application starts by creating a new **Vue instance** with the `Vue` function:
+Setiap aplikasi Vue dimulai dengan membuat **Permintaan Vue** dengan fungsi `Vue` seperti berikut :
 
 ```js
 var vm = new Vue({
-  // options
+  // opsi
 })
 ```
 
-Although not strictly associated with the [MVVM pattern](https://en.wikipedia.org/wiki/Model_View_ViewModel), Vue's design was partly inspired by it. As a convention, we often use the variable `vm` (short for ViewModel) to refer to our Vue instance.
+Meskipun tidak dengan terkait dengan ketat dengan [Pola MVVM](https://en.wikipedia.org/wiki/Model_View_ViewModel), desain yang dimiliki Vue sebagian terinspirasi oleh konsep tersebut. Sebagai ketentuan, kita kadang menggunakan variabel `vm` ( kependekan dari ViewModel ) untuk mengacu pada permintaan Vue kami.
 
-When you create a Vue instance, you pass in an **options object**. The majority of this guide describes how you can use these options to create your desired behavior. For reference, you can also browse the full list of options in the [API reference](../api/#Options-Data).
 
-A Vue application consists of a **root Vue instance** created with `new Vue`, optionally organized into a tree of nested, reusable components. For example, a todo app's component tree might look like this:
+Ketika anda membuat permintaan Vue, anda mengirimkan **objek pilihan**. Sebagian besar dari panduan ini menjelaskan bagaimana anda bisa menggunakan opsi ini untuk membuat tindakan yang anda inginkan. Sebagai contoh, anda juga bisa melihat lihat daftar lengkap dari opsi ini di [Contoh API](../api/#Options-Data).
+
+
+Sebuah aplikasi Vue terdiri dari **permintaan dasar Vue** yang dibuat dengan `new Vue`, secara opsional di atur ke dalam struktur pohon, komponen yang bisa digunakan ulang. Sebagai contoh, komponen aplikasi pengingat mungkin terliat seperti ini :
```

Comment:
**root Vue instance** -> **akar (root) Vue instance**
tree of nested gak usah di terjemahkan, pake italic aja.
terliat -> terlihat